README.md

```diff
@@ -118,9 +118,10 @@ The Message Service sends messages to users and recipients for the following rea
 - Providing one-time passcodes (OTPs) to recipients
 - Sending emails to users during account creation and account recovery flows
 
-Note that the Message Service requires that both SMS and email services are configured. For emails, AWS SES is supported. For SMS messages to recipients, Twilio is supported. AWS SNS support is not integrated yet.
+Note that the Message Service requires that both SMS and email services are configured. For emails, AWS SES and Twilio Sendgrid are supported. For SMS messages to recipients, Twilio is supported. AWS SNS support is not integrated yet.
```

Comment:
AWS SNS is supposed to work, right? I believe this phrase is outdated.

Reply:
Addressed in [50bdd62](https://github.com/stellar/stellar-disbursement-platform-backend/pull/444/commits/50bdd621e9fb9806e27310a11d4ce411b09414be)